jjb/mkck/mkck-trigger.yaml

```diff
@@ -506,6 +515,37 @@
             properties-file: 'target.properties'
         - shell: "git clone https://github.com/suse/sesci -b $CI_BRANCH"
         - create-sesci-venv
+        - multijob:
+            name: Deliver artifacts
+            #condition: ALWAYS
+            condition: SUCCESSFUL
+            projects:
+                - name: storage-deliver-mkck-artifact
+                  current-parameters: true
+                  abort-all-job: true
+                  predefined-parameters: |
+                    DIST_CEPH={dist_ceph}
+                    SITE=ovh.net
```

Comment:
should `SITE` be configurable to be able to run it on ECP?

Reply:
Yes, it should be configurable but I don't remember exactly that copy-artifacts could render a placeholder with parameter for predefined-parameters, thus, I just left it here, we can create another patch to do this.

Reply:
ok